fabfile/config.py

```diff
@@ -14,7 +14,7 @@
 CITUS_INSTALLATION = os.path.join(HOME_DIR, 'citus-installation')
 
 PG_VERSION = '9.6.1'
-PG_CONFIGURE_FLAGS = ['--with-openssl']
+PG_CONFIGURE_FLAGS = ['--with-openssl', '--with-uuid=e2fs']
```

Comment:
This will be a problem for any part where we install citus and run tests. I feel like we should try to find some other way about this extension problem. Installing it everywhere does not seem like a good solution to me. Maybe you should start a discussion in team?

Reply:
Then I will customize adding this flag for valgrind test. Other than this, I plan to add an optional output to regression outputs

Reply:
Removed related commit from this branch and customized it on https://github.com/citusdata/test-automation/pull/130

Reply:
Yeah I think the ideal solution would be to have optional output in regression tests, I dont know how easy that is.  Because this is adding a dependency to all of our tools etc that use tests.

Reply:
Yep, I will add optional output for this soon